### PR TITLE
Event poster

### PR DESCRIPTION
**Pull request by adrienz1**

### CODE REVIEW

**Comment by adrienz1** [comment]
Screenshot_20240401_162508
Screenshot_20240401_162508

**Comment by adrienz1** [comment]
Screenshot_20240401_162508
Screenshot_20240401_162535

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java [hunk 1]
```diff
@@ -30,7 +30,7 @@
  */
 public class OrgEventFragment extends Fragment {
 
-    private ImageView qrCodeImage;
+    private ImageView eventPoster;
```
Comment (dpardell) [inline code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java:33]: Wait, so are we no longer showing the event check in qr code?

**Review by dpardell — COMMENTED** [review]
Quick question about what happens to the event check in QR code

File: code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java (hunk 1 above)
Comment (adrienz1) [reply code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java:33]: no, since we have the share function

Comment (dpardell) [reply code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java:33]: True, but currently the "Promote" button only gives us the promotional QR code. We also need to be able to share the event check in QR code.

Comment (adrienz1) [reply code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java:33]: I thought that the ´Sharé’ button shared the check in qr

Comment (dpardell) [reply code/EventLinkQR/app/src/main/java/com/example/eventlinkqr/OrgEventFragment.java:33]: Oh wait, you are so right. my bad

**Review by dpardell — APPROVED** [review]

**Review by Shubjaura — APPROVED** [review]
Looks good to me.

**Pull request merged**